lambdas/backend/express-route-handlers.js

```diff
@@ -492,7 +492,7 @@ async function postAdminCatalogVisibility(req, res) {
         // try {
```

Comment:
Can you uncomment this try/catch block and respond with `res.status(500).json({ error: 'Internal Server Error' })`? Currently, the lambda runs until it times out when this edge case is hit.

Reply:
Done